acceptance-amazon-rds-sync/rds-common

```diff
@@ -11,6 +11,34 @@ REGEX_DATE="<SnapshotCreateTime>(.*?)T(.*?)Z</SnapshotCreateTime>"
 #
 # Common RDS functions
 #
+rds_get_latest_snapshot() {
+	latest_snapshot=
+	SNAPSHOTS=$(rds-describe-db-snapshots -i "$SNAPSHOT_INSTANCE");
+
+	while read -r line; do
+		stringarray=($line);
+		snapshot_name=${stringarray[1]};
+
+    # automatic backups named as rds:dbs3-2015-03-02-01-11
+    REGEX="rds:dbs3-[0-9]{4}-[0-9]{2}-[0-9]{2}"
+    if ! [[ "$snapshot_name" =~ $REGEX ]]; then
+    	echo "No automatic backup, $snapshot_name"
+    	continue
+    fi
+
+    # naively find latest creation date
+    if [[ -z $latest_snapshot ]]; then
+    	latest_snapshot=$snapshot_name
+    elif [[ "$snapshot_name" > "$latest_snapshot" ]]; then
```

Comment:
ps.: please remove the unused SNAPSHOT 'constant' from [config.template](acceptance-amazon-rds-sync/config.template) and update the README.md accordingly ?